python/F3DPythonBindings.cxx

```diff
@@ -340,8 +340,42 @@ PYBIND11_MODULE(pyf3d, module)
     .def_static("load_plugin", &f3d::engine::loadPlugin, "Load a plugin")
     .def_static(
       "autoload_plugins", &f3d::engine::autoloadPlugins, "Automatically load internal plugins")
-    .def_static("get_plugins_list", &f3d::engine::getPluginsList);
+    .def_static("get_plugins_list", &f3d::engine::getPluginsList)
+    .def_static("get_lib_info", &f3d::engine::getLibInfo, py::return_value_policy::reference)
+    .def_static("get_readers_info", &f3d::engine::getReadersInfo);
 
+  // libInformation
+  py::class_<f3d::engine::libInformation>(module, "LibInformation")
+    .def(py::init<>())
+    .def(py::init<const std::string&, const std::string&, const std::string&, const std::string&,
+                  const std::string&, const std::string&, const std::string&, const std::string&,
+                  const std::string&, const std::string&, const std::string&, const std::string&,
+                  const std::string&>())
```

Comment:
These object will only be instantiated by the library itself and not from any python code so exposing constructors shouldn't be necessary